config/initializers/devise.rb

```diff
@@ -1,13 +1,16 @@
 require 'saml_idp_constants'
+require_relative '../../app/mailers/concerns/mailable'
```

Comment:
I was hoping we wouldn't need these `require_relative` lines because `app/*/concerns` should be on the autoload paths according to this Rails guide:

Source: http://guides.rubyonrails.org/autoloading_and_reloading_constants.html#autoload-paths

> ...Rails has a collection of directories similar to $LOAD_PATH in which to look up post.rb. That collection is called autoload_paths and by default it contains:
>
> - Any existing second level directories called `app/*/concerns` in the application and engines.

But after checking out this branch and messing around, `app/mailers/concerns` was not end up on the autoload path inside the initializer :expressionless:


Long story short....would you be open to adding this line to `application.rb` so we can remove this `require_relative`?

```diff
diff --git a/config/application.rb b/config/application.rb
index 7c9e917..b2b4972 100644
--- a/config/application.rb
+++ b/config/application.rb
@@ -29,6 +29,8 @@ module Upaya
 
     config.middleware.use Rack::Attack
 
+    config.autoload_paths << Rails.root.join("app/mailers/concerns")
+
     # Configure Browserify to use babelify to compile ES6
     config.browserify_rails.commandline_options = '-t [ babelify --presets [ es2015 ] ]'
   end
```